### DISCUSSION TITLE
Using Hotspot, is my use case unique??  DHCP and IP Address assignment

### DISCUSSION THREAD

**Discussion by mikpeter99** [MichaIng/DietPi#6749] [Q&A]
Successful install of DietPi, trying to config hotspot.   Trying to use ensure that i get DHCP from main router, is there anyway to force that like transparent bridge mode?   Here is the issue, i use airplay with sonos and other media type devices, want to make sure all the devices are on the same subnet so that they are recognized each other.   I am really trying to make this an wifiextender more than anything else!   Thougths

**Comment by MichaIng** [comment]
WiFi Hotspot currently sets up the Pi as NAT. Coincidentally we talked just on our last meeting about the use case of a bridge, e.g. also if one wants to access WiFi clients from the Pi <-> router side network.

However, currently a bridge needs to be setup manually. Disable the DHCP server for this and configure the bridge in `/etc/network/interfaces`.  Here is a very simple example: https://stackoverflow.com/questions/51100710/how-to-get-bridge-support-with-etc-network-interfaces

@StephanStS is experimenting which such things and will probably write an article about it, how to setup this best on a DietPi system.

**Reply by mikpeter99** [reply]
Going to give it a try!!!   Wow, thanks for the quick reply and turnaround with ideas!!!  Ill let you know.   MP
